easybuild/easyconfigs/r/rgeos/rgeos-0.5-1-foss-2019a-R-3.6.0.eb

```diff
@@ -10,7 +10,7 @@ description = """R interface to Geometry Engine - Open Source (GEOS) using the C
 
 toolchain = {'name': 'foss', 'version': '2019a'}
 
-source_urls = ['https://cran.r-project.org/src/contrib/']
+source_urls = ['https://cran.r-project.org/src/contrib/Archive/rgeos']
```

Comment:
@arielzn We should keep both URLs, to avoid running into this issue again with future versions of `rgeos`?

Also, please apply the same fix to the other `rgeos` easyconfigs (especially since it's trivial to verify the fix via `eb --fetch`)

Reply:
ok, i'll add for all.

I think once there's a new version, any archived stays forever in `/contrib/Archive`.
but for consistency probably makes more sense to have all R packages with both urls.